dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1027,7 +1027,7 @@ def ones_like(x1, dtype=None, order='C', subok=False, shape=None):
         _shape = shape if shape is not None else x1.shape
         _dtype = dtype if dtype is not None else x1.dtype
 
-        return dpnp_init_val(_shape, _dtype, 1)
+        return dpnp_ones(_shape, _dtype)
```

Comment:
We also need to have kernels for the functions `ones_like` and `zeros_like`. Better to add the kernels in separate PR.